views/includes/download_chooser.pug

```diff
@@ -6,9 +6,9 @@ fieldset.download-links
   p
     button.btn.btn-default(type='submit' formaction='download_full_not_fillable_fr') Requirements with informative sections (French)
   p
-    button.btn.btn-default(type='submit' formaction='download_evaluation_en') Evaluation template (English)
+    button.btn.btn-default(type='submit' formaction='download_evaluation_en') Testable clauses (English)
```

Comment:
I see what you mean that this wording is awkward. Can you maybe change this to be "list of testable clauses".